recipies/ScallopedPotato.html

```diff
@@ -8,5 +8,26 @@
 </head>
 <body>
   <h1> Scalloped Potato </h1>
+  <img src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimages.media-allrecipes.com%2Fuserphotos%2F6067206.jpg&w=596&h=792&c=sc&poi=face&q=85" alt="">
+<h3>Description</h3>
+<body>
+ <p> Scalloped Potatoes are an easy classic recipe, perfect for your Easter dinner, Christmas, Thanksgiving or even just for Sunday dinner.</p>
+
+ <p>In this side dish, thinly sliced potatoes and onions are layered in an easy homemade cream sauce and baked until tender, golden, and bubbly. Potato perfection!</p> 
+</body>
```

Comment:
There is another body tag inside your main body tag.
- [ ] HTML document should contain one body